actionpack/lib/action_dispatch/middleware/flash.rb

```diff
@@ -177,13 +177,6 @@ def key?(name)
         @flashes.key? name.to_s
       end
 
-      def delete(key)
```

Comment:
Why did the method have to move?

Reply:
Because `delete` and `discard` are similar, I think they should be declared close together.

Reply:
I would consider this change to be [cosmetic](https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#write-your-code), which won't be accepted. So if there is not a good reason to move it otherwise, I think it should be reverted.

Reply:
Thanks for the tip. Moved back to the original position.